scripts/zkai_d128_gate_value_activation_fused_gate.py

```
@@ -0,0 +1,549 @@
+
+def canonical_bytes(value: Any) -> bytes:
+    return json.dumps(value, sort_keys=True, separators=(",", ":")).encode("utf-8")
+
+
+def payload_commitment(payload: dict[str, Any]) -> str:
+    canonical = copy.deepcopy(payload)
+    canonical.pop("payload_commitment", None)
+    return "sha256:" + hashlib.sha256(canonical_bytes(canonical)).hexdigest()
+
+
+def read_json(path: pathlib.Path, max_bytes: int, label: str) -> Any:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `read_json`

(ANN401)